x-pack/plugins/case/server/routes/api/cases/patch_cases.ts

```diff
@@ -76,16 +82,25 @@ export function initPatchCasesApi({
               .join(', ')} has been updated. Please refresh before saving additional updates.`
           );
         }
-        const updateCases: CasePatchRequest[] = query.cases.map((thisCase) => {
+        const updateCases: ESCasePatchRequest[] = query.cases.map((thisCase) => {
```

Comment:
To avoid casting we need to write it like that
```
const updateCases: ESCasePatchRequest[] = query.cases.map((updateCase) => {
          const currentCase = myCases.saved_objects.find((c) => c.id === thisCase.id);
          const { connector, ...thisCase } = updateCase;
          return currentCase != null
            ? getCaseToUpdate(currentCase.attributes, {
                ...thisCase,
                ...(connector != null
                  ? { connector: transformCaseConnectorToEsConnector(connector) }
                  : {}),
              })
            : { id: thisCase.id, version: thisCase.version };
        });
```

Reply:
I might be wrong but I think we need to convert the connector attribute with `transformESConnectorToCaseConnector` when we return`returnUpdatedCase`. Can we check that?